electron-app/src/main/index.ts

```diff
@@ -377,20 +481,28 @@ class LlamaFarmApp {
           console.warn('Services may not have started properly. Status:', verifyResult.stdout)
         }
 
-        // Success - exit retry loop
+        // Success - log version info and exit retry loop
+        const cliVersion = await this.getCLIVersion()
+        const sourceVersion = await this.getSourceVersion()
+        console.log(`Services started successfully. CLI version: ${cliVersion || 'unknown'}, Source version: ${sourceVersion || 'unknown'}`)
         return
       } catch (error) {
         const errorMsg = error instanceof Error ? error.message : String(error)
         console.error(`Service start attempt ${attempt} failed:`, errorMsg)
 
         if (attempt >= maxRetries) {
+          // Include version info in error for debugging
+          const cliVersion = await this.getCLIVersion()
+          const sourceVersion = await this.getSourceVersion()
+          const versionInfo = `(CLI: ${cliVersion || 'unknown'}, Source: ${sourceVersion || 'unknown'})`
```

Comment:
We could probably refactor to prevent duplicate calls to getCLIVersion and getSourceVersion